public/search-api/frameworks.php

```diff
@@ -15,6 +15,9 @@
 
 // Set empty vars
 $dataToReturn = [];
+$allStatus = ['Live', 'Expired - Data Still Received', 'Future (Pipeline)', 'Planned (Pipeline)', 'Underway (Pipeline)', 'Awarded (Pipeline)'];
```

Comment:
Can we do this as an enum or some other format so we aren't duplicating these strings all over the place?